cupy/core/_carray.pxd

```diff
@@ -1,4 +1,5 @@
 cimport cython  # NOQA
+from libcpp cimport bool
```

Comment:
Interesting...Didn't know `libcpp.bool`🙂 Usually we just use the Cython built-in `bint`. While I like `bool` better, I wonder if it'd conflict with Python's `bool`, which is still respected in Cython? Just curious.

Reply:
I am thinking to actually change it to `bint` or rename the import with `as cbool`